DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -132,7 +132,9 @@
     <![CDATA[superClusterRelinked_.reset();]]>
   </ioread>
 
-  <class name="pat::Jet"  ClassVersion="14">
+  <class name="pat::Jet"  ClassVersion="16">
+   <version ClassVersion="16" checksum="727883729"/>
+   <version ClassVersion="15" checksum="3059757499"/>
```

Comment:
I had 15 with the FwdPtrs, when I updated it screwed everything up and I had to update to 16 to make it work.